--- contrib/depends/packages/boost.mk
+++ contrib/depends/packages/boost.mk
@@ -30,6 +30,7 @@
 define $(package)_preprocess_cmds
   patch -p1 < $($(package)_patch_dir)/fix_aroptions.patch &&\
   patch -p1 < $($(package)_patch_dir)/fix_arm_arch.patch &&\
+  sed -i.bak '/-fcoalesce-templates/d' tools/build/src/tools/darwin.jam tools/build/src/tools/darwin.py && rm -f tools/build/src/tools/darwin.jam.bak tools/build/src/tools/darwin.py.bak &&\
   echo "using $(boost_toolset_$(host_os)) : : $($(package)_cxx) : <cxxflags>\"$($(package)_cxxflags) $($(package)_cppflags)\" <linkflags>\"$($(package)_ldflags)\" <archiver>\"$(boost_archiver_$(host_os))\" <arflags>\"$($(package)_arflags)\" <striper>\"$(host_STRIP)\"  <ranlib>\"$(host_RANLIB)\" <rc>\"$(host_WINDRES)\" : ;" > user-config.jam
 endef
 

