--- a/src/cryptonote_basic/miner.cpp
+++ b/src/cryptonote_basic/miner.cpp
@@ -47,9 +47,11 @@
 #ifdef __APPLE__
   #include <sys/times.h>
-  #include <IOKit/IOKitLib.h>
-  #include <IOKit/ps/IOPSKeys.h>
-  #include <IOKit/ps/IOPowerSources.h>
   #include <mach/mach_host.h>
   #include <AvailabilityMacros.h>
   #include <TargetConditionals.h>
+#if TARGET_OS_OSX
+  #include <IOKit/IOKitLib.h>
+  #include <IOKit/ps/IOPSKeys.h>
+  #include <IOKit/ps/IOPowerSources.h>
+#endif
 #elif defined(__linux__)
